packages/content-common/src/types.ts

```diff
@@ -115,6 +115,19 @@ export type CreateCustomSectionApiInput = {
   disabled: boolean;
 };
 
+export type UpdateCustomSectionApiInput = {
```

Comment:
Are you using this type anywhere? It should be used in the resolver level integration tests